features/step_definitions/social_media_steps.rb

```diff
@@ -48,5 +48,5 @@
     social_container = Organisation.last
     visit organisation_path(social_container)
   end
-  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "Connect with #{social_container.display_name} on #{title}")
+  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "#{title}")
```

Comment:
Similar to previous:

```suggestion
  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: title)
```